### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by matanox** [mkdocs/mkdocs#3778] [Q&A] [answered]
Hi,

I'm sorry to see about the project's team decline on the April post.

I have followed the getting started guide, with the aim of building api docs out of only select sources. I could not find a way to specify only those sources, and either way ended up (by following the getting started) with a static website documenting mkdocs itself. 

How do we tell mkdocs where the pythonn sources to be documentd are?

Thanks,
Matan

**Comment by pawamoy** [comment, marked as answer]
I think you're looking for the mkdocstrings plugin.

**Comment by matanox** [comment]
Oh, thank you! looks like what I was looking for indeed! 
So what's the use case for mkdocs without that plugin? I must have misunderstood it!

**Reply by pawamoy** [reply]
The use case is documenting various things that are not always code :)

**Comment by matanox** [comment]
Thank you very much. I'm now up and running with an mkdocs server updating and building markdown pages automatically generated from specific python sources using mkdocstrings. 

Just one more for getting started. Any advice for gently porting an existing suite of interlinking markdown pages that I already have? Currently they are just part of my git repo and browseable on the Github repo page as such starting from a `readme.md`. 

What would be a typical course of action making them in any way better by porting them to be mkdocs managed? Re-arranging my repo structure to have them under the mkdocs directory and deploying them to Github Pages?

**Reply by pawamoy** [reply]
Hmmm, I don't really have advice no, apart from:

- put your Markdown documents into `docs`
- rearrange as you see fit
- fix any relative links (MkDocs will tell you which ones)
